net/pfSense-pkg-Avahi/files/usr/local/pkg/avahi/avahi.inc

```diff
@@ -0,0 +1,128 @@
+<?php
+/*
+ * avahi.inc
+ *
+ * part of pfSense (https://www.pfsense.org)
+ * Copyright (c) 2018 Denny Page
```

Comment:
Don't remove Rubicon Copyright

Reply:
I didn't remove the copyright--in effect it wasn't there to begin with. This package is a complete ground up writing, unrelated to the prior package. This particular file was written completely from scratch.